source/Junkership.js

```diff
@@ -83,9 +88,10 @@ export default class Junkership extends Pixi.Sprite {
             }
         }
     }
-    
+
     onCollision(collidedWith) {
         game.removeChild(this)
+        Reference.ControlScheme.keys[this.controls].inUse = false
```

Comment:
Yep. Otherwise, hitting any of the controls for the ship will make it spawn another ship with the same control scheme, leaving you with 4 ships stacked on top of each other.